datadog_sync/commands/shared/options.py

```diff
@@ -199,6 +203,29 @@ def click_config_file_provider(ctx: Context, opts: CustomOptionClass, value: Non
 ]
 
 
+_sync_common_options = [
+    option(
+        "--force-missing-dependencies",
+        required=False,
+        is_flag=True,
+        default=False,
+        show_default=True,
+        help="Force importing and syncing resources that could be potential dependencies to the requested resources.",
+        cls=CustomOptionClass,
+    ),
+    option(
+        "--create-global-downtime",
+        required=False,
+        is_flag=True,
+        default=False,
```

Comment:
we should probably switch the create-global-downtime to default to true since we almost always want it to happen.